File: Selenium/Project/Activity2.py
"""Get the url of the header image 
Goal: Print the url of the header image to the console 
a. Open a browser. 
b. Navigate to ‘http://alchemy.hguy.co/crm’.  
c. Get the url of the header image. 
d. Print the url to the console. 
e. Close the browser."""

#Import the webdriver
from selenium import webdriver
from selenium.webdriver.common.by import By

#Initialise the driver
with webdriver.Firefox() as driver:
    #Open & navigate to crm browser
    driver.get("http:///alchemy.hguy.co/crm")

    #Find the URL of the header image
    #headerImageURL = driver.find_element(By.XPATH, "//img[contains(text(), 'company_logo')]")
    headerImageURL = driver.find_element(By.XPATH, "//img[contains(@src, 'company_logo')]")
    #Print the url of the header image
    print("URL of the header image: ", headerImageURL.get_attribute("src"))

File: Selenium/Project/Activity2.java
/*
a. Open a browser. 
b. Navigate to ‘http://alchemy.hguy.co/crm’. 
c. Get the url of the header image. 
d. Print the url to the console. 
e. Close the browser. 
 */

package crmactivities;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;
import org.testng.annotations.AfterClass;


public class Activity2 {
	WebDriver driver;
	WebDriverWait wait;
	
	@BeforeClass
	public void setUp() {
		driver = new FirefoxDriver();
		wait = new WebDriverWait(driver, Duration.ofSeconds(10));
		
		driver.get("http://alchemy.hguy.co/crm");
	}
	
	@Test
	public void printURL() {
		//Find the url of the headerimage
		WebElement headerImageURL = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));
		
		
		//Print the url to the console
		System.out.println("Header image URL is " + headerImageURL);
	}
	
	@AfterClass
	public void closeBrowser() {
		driver.close();
	}
}
